### DISCUSSION TITLE
Show HTTP Response section on Log Entry Record Page for Apex Continuation

### DISCUSSION THREAD

**Discussion by BastienRaoul** [jongpie/NebulaLogger#586] [Ideas]
Hello @jongpie, 

For one of our older processes, we would like to log the HTTP response of an apex continuation.

The problem we face is that a continuation always returns a null response status.
In the Log Entry Record Page, the HTTP response section is only visible if the response status is not null, therefore we cannot see the section even if the others fields of the HTTP response are not null.

Would it be possible to improve this by creating a dedicated field that is set when at least one of the HTTP response field is not null and use this field for the section visibility ?

Thank you,
Bastien

**Comment by jongpie** [comment]
@BastienRaoul this is great feedback, I haven't used `Continuation` much in Apex, so thanks for your insights! I've added a change for this in ~PR #613~ PR #617 - now, the HTTP Response section will show up if any of these 3  conditions are `true`:
- `HttpResponseStatus__c != null` (existing check)
- `HasHttpResponseBody__c == true`
- `HasHttpResponseHeaderKeys__c == true`

These are existing fields, so I'm hoping that checking for 1 of them to be `true` will be enough. If this still doesn't quite work correctly, I can add a new field in a future release.

**Reply by jongpie** [reply]
@BastienRaoul this has been released in [`v4.12.8`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.12.8)
